build.sh

```diff
@@ -227,15 +227,22 @@ fi
 
 # If `./build.sh cuml` is called, don't build C/C++ components
 if completeBuild || hasArg libcuml || hasArg prims || hasArg bench || hasArg cpp-mgtests; then
-    # If there are no targets specified when calling build.sh, it will
-    # just call `make -j`. This avoids a lot of extra printing
     cd ${LIBCUML_BUILD_DIR}
-    make -j${PARALLEL_LEVEL} ${MAKE_TARGETS} VERBOSE=${VERBOSE} ${INSTALL_TARGET}
+    build_args="--target ${MAKE_TARGETS} ${INSTALL_TARGET}"
```

Comment:
I can't remember what is the source of `${INSTALL_TARGET}` but isn't it just the word `install`? What do you think of removing it and just using `build_args="--target ${MAKE_TARGETS} install"` which is a bit easier to follow

Reply:
At [line 127](https://github.com/rapidsai/cuml/pull/3487/files#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R126-R127), we can potentially suppress installation with the `-n` flag by setting `INSTALL_TARGET=""`, so I think we still need it, unfortunately.